### DISCUSSION TITLE
Bootloop

### DISCUSSION THREAD

**Discussion by hphp22f** [bmax121/APatch#365] [Q&A] [answered]
When device is bootloop, then how to remove any module useing custom recovery.

**Comment by masteruni** [comment]
Did you manage to backup your boot via recovery before flashing the patched boot.img?

**Comment by marat2509** [comment, marked as answer]
You can mount /data/adb/ap/modules.img, and from the mounted image you can delete the module you need. 

Another option, but more crude, is to delete or rename this file, which will unmount all modules at boot time
